GliaWidgets/Lib/File/LocalFile.swift

```diff
@@ -2,7 +2,15 @@ import UIKit
 
 class LocalFile {
```

Comment:
I don't know why it is the way it is, but there are no newlines between all the multiline `lazy var`s in this file. Please insert them for readability purposes (except for after `fileExtension`, as it fits in a single line).